dom.bs

```diff
@@ -6656,29 +6669,53 @@ string <var>namespace</var> (default null):</p>
 
  <li><p>Otherwise, <a lt="append an attribute">append</a> <var>attr</var> to <var>element</var>.
 
+ <li><p>Set <var>attr</var>'s <a for=Attr>value</a> to <var>verifiedValue</var>.
+
  <li><p>Return <var>oldAttr</var>.
 </ol>
 </div>
 
 <div algorithm>
-<p>To <dfn export id=concept-element-attributes-set-value>set an attribute value</dfn> given an
-<a for=/>element</a> <var>element</var>, a string <var>localName</var>, a string <var>value</var>,
-an optional null or string <var>prefix</var> (default null), and an optional null or string
-<var>namespace</var> (default null):
+<p>To <dfn export id=concept-element-attributes-set-value>set an attribute value</dfn> given an <a
+for=/>element</a> <var>element</var>, a string <var>localName</var>, a string or {{TrustedType}}
+<var>value</var>, an optional null or string <var>prefix</var> (default null), an optional null or
+string <var>namespace</var> (default null), and an optional boolean <var>verify</var> (default
+false):
 
 <ol>
  <li>Let <var>attribute</var> be the result of
  <a lt="get an attribute by namespace and local name">getting an attribute</a> given
  <var>namespace</var>, <var>localName</var>, and <var>element</var>.
 
- <li>If <var>attribute</var> is null, create an <a>attribute</a> whose <a for=Attr>namespace</a> is
- <var>namespace</var>, <a for=Attr>namespace prefix</a> is <var>prefix</var>,
- <a for=Attr>local name</a> is <var>localName</var>, <a for=Attr>value</a> is <var>value</var>, and
- <a for=Node>node document</a> is <var>element</var>'s <a for=Node>node document</a>, then
- <a lt="append an attribute">append</a> this <a>attribute</a> to <var>element</var>, and then
- return.
+ <li><p>Let <var>attributeExists</var> be false if <var>attribute</var> is null, and true otherwise.
+
+ <li><p>If <var>attributeExists</var> is false, set <var>attribute</var> to an <a>attribute</a>
+ whose <a for=Attr>namespace</a> is <var>namespace</var>, <a for=Attr>namespace prefix</a> is
+ <var>prefix</var>, <a for=Attr>local name</a> is <var>localName</var>, <a for=Attr>value</a> is
+ <var>value</var>, and <a for=Node>node document</a> is <var>element</var>'s <a for=Node>node
+ document</a>.
+
+ <li><p>Let <var>verifiedValue</var> be <var>value</var>.
+
+ <li><p>If <var>verify</var> is true:
+  <ol>
+   <li><p>Set <var>verifiedValue</var> to the result of calling <a>verify attribute value</a>
+   <var>value</var> for <var>attribute</var>, with <var>element</var>.
+
+   <li><p>Set <var>attributeExists</var> to true if <var>element</var> <a lt="has an attribute">has
```

Comment:
No newlines in phrasing level elements.

Reply:
I think I've addressed all of these, will check the github diff to make sure